scripts/scripts.js

```diff
@@ -709,6 +727,7 @@ function loadDelayed() {
  * Decorates the page.
  */
 async function loadPage(doc) {
+  loadFeedData();
```

Comment:
Do we have to do this globally? Won't it be enough to do this in the feed block

Reply:
ah yes, indeed! updated in [ab99b0d](https://github.com/adobe/helix-website/pull/537/commits/ab99b0d3bb6c0ee5c90a7568fea6ad43e426249d)